portal-web/test/functional/com/liferay/portalweb/macros/Segmentation.macro

```diff
@@ -142,11 +124,6 @@ definition {
 
 	macro clickToReEnableSegmentsWarningLink {
 		Click(locator1 = "Segmentation#ENABLE_SEGMENTS_INSTANCE_SETTINGS");
-
-		AssertVisible(
-			key_configurationName = "Segments Service",
-			key_configurationScope = "Virtual Instance Scope",
-			locator1 = "SystemSettings#SCOPED_CONFIGURATION_NAME");
 	}
 
 	macro closeSegmentationDisabledWarning {
```

Comment:
This macro is being used in some tests, and maybe there's no need for it. I created it, but it makes to assertions (one direct and another using a macro), when the goal should be only to close the alert. Do you think we should get rid of it?

Reply:
Or maybe keep as a macro, but just with the click.